libnavui-androidauto/src/main/java/com/mapbox/androidauto/car/navigation/speedlimit/CarSpeedLimitRenderer.kt

```diff
@@ -46,39 +52,52 @@ class CarSpeedLimitRenderer(
         }
     }
 
+    private var scope: CoroutineScope? = null
```

Comment:
```suggestion
    private lateinit var scope: CoroutineScope
```

This does not need to be null right?

Reply:
Probably not, I just prefer to nullify references, when objects are no longer necessary. But I doubt anything can leak through the scope, so I changed to `lateinit var`.